R/DTW.R

```diff
@@ -14,8 +14,8 @@
 #'   evaluation grid for the second QTS.
 #' @param distance_only A boolean specifyung whether to only compute distance
 #'   (no backtrack, faster). Default is \code{FALSE}.
-#' @param step_pattern The choice for the step pattern of the warping path (see
-#'  \code{\link[dtw]{stepPattern}} for more information)
+#' @param step_pattern The choice for the step pattern of the warping path. (default to \code{dtw::symmetric2},
+#' which is symmetric, normalizable with no local slope constraints. See \code{\link[dtw]{stepPattern}} for more information).
```

Comment:
You need to re-indent documentation to avoid lengthy lines. Also please write proper sentences. No parentheses. `default` should start with capital letter and ends with an `s` (it is a verb). Explanation about what `symmetric2` does is not clear enough because it seems identical to `symmetric1` with the explanation you provide.

Reply:
```symmetric1``` is not normalizable and is presented as "quasi symetric" in dtw::stepPattern